src/Kraken.cpp

```diff
@@ -7,14 +7,14 @@
 
 #include <czmq.h>
 #include <g2log.hpp>
-
 #include "Kraken.h"
 
 
 /// Constructing the server/Kraken that is about to be connected/impaled by the client/Harpoon
 Kraken::Kraken():
 mLocation(""), 
-mQueueLength(10),
+mQueueLength(1),
+mMaxChunkSize(10485760), //10MB
```

Comment:
I'm confused with this. 
On line 17 you set it to 10MB, but in the function ```MaxChunkSizeInBytes``` you can set it to something else. Why would you want to set it to something else?